libraries/apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/Adapters.kt

```diff
@@ -117,6 +118,22 @@ class ApolloOptionalAdapter<T>(private val wrappedAdapter: Adapter<T>) : Adapter
   }
 }
 
+@JvmName("-obj")
+fun <T> Adapter<T>.obj() = ObjectAdapter(this)
```

Comment:
Input objects are back to being `Adapter` instead of `CompositeAdapter`. This way, all input types implement  the `Adapter` interface. Also they were not using anything in `AdapterContext`